images/bootstrap/runner.sh

```diff
@@ -100,7 +100,14 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
 fi
 
 # Use a reproducible build date based on the most recent git commit timestamp.
-export SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct || true)
+SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct || true)
+export SOURCE_DATE_EPOCH
+
+# TODO(bentheelder): this is a band-aid! Delete this when we've exposed this data
+# in the pod-utils first-class.
+if [[ "${METADATA_BANDAID:-false}" == "true" ]]; then
+    echo "{\"job-version\": \"$(git rev-parse HEAD)\"}" >> "${ARTIFACTS}/metadata.json"
```

Comment:
does anything else currently write a `metadata.json` file?

Reply:
kubetest does (don't flip it for e2e jobs :-) )

Reply:
yes, otherwise i'd have just enabled this by default :|

Reply:
OTOH enabled not by default also means we can easily find everything using it when we go to remove it